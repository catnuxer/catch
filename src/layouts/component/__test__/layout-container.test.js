import React from "react";
import serializer from "jest-emotion";
import Container from "../layout-container";
import renderer from "react-test-renderer";

expect.addSnapshotSerializer(serializer);

describe("layout-container test", () => {
  it("snappshoot", () => {
    expect(
      renderer.create(<Container>Test</Container>).toJSON()
    ).toMatchSnapshot();
  });
});
