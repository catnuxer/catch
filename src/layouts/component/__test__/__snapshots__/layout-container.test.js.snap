// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout-container test snappshoot 1`] = `
.emotion-0 {
  margin: 0px 80px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

@media only screen and (min-width :768px) and (max-width :1024px) {
  .emotion-0 {
    margin: 0px 30px;
  }
}

@media only screen and (min-width :320px) and (max-width :768px) {
  .emotion-0 {
    margin: 0px 15px;
  }
}

<div
  className="emotion-0"
>
  Test
</div>
`;
